Upwdated WEADME.md (#194)
replugged-org/replugged: README.md

# Replugged

> Maintained fork of [powercord](https://github.com/powercord-org/powercord) - a lightweight [@discord](https://github.com/discord) client mod focused on simplicity and performance. https://replugged.dev/

Reviewer: Discord link is not needed, website link is on the right already

## Installation/Uninstallation

> See the [installation page of the Replugged Wiki](https://github.com/replugged-org/replugged/wiki/Installation).

Reviewer: Let's change this to the installation page on the website

### Plugins/Themes

> Join our [official Discord server](https://discord.gg/replugged) and scroll through the channels to find new supported and maintained plugins or themes. 

Reviewer: ```suggestion
Plugins and themes can be found in our [official Discord server](https://discord.gg/replugged) in #plugin-links and #theme-links, respectively. 
```

[![Replugged Discord Server](https://discordapp.com/api/guilds/1000926524452647132/widget.png?style=banner2)](https://discord.gg/replugged)

> If the plugin/theme you want isn't there, check out the old archived [Powercord server](https://discord.gg/powercord) and you might find plugins and themes there that might work. But most of them will probably be outdated. 

Reviewer: ```suggestion
If the plugin/theme you want isn't there, check out the old archived [Powercord server](https://discord.gg/powercord). Note that bc some of the plugins may be outdated or abandoned. 
```

#### How to install plugins/themes

> Join our [official Discord server](https://discord.gg/replugged) or go to the GitHub repository of the plugin or theme where they might have installation instructions. Most likely they'll have a link that installs it automatically. The plugin/theme channel in the Discord server also has an install button which you can conviently click!

Reviewer: This needs to be reworded

# Replugged
Maintained fork of [powercord](https://github.com/powercord-org/powercord) - a lightweight Discord client mod focused on simplicity and performance. https://replugged.dev/

<br><hr>

> ## Installation/Uninstallation

Reviewer: ```suggestion
## Installation
```

See the [installation page of the Replugged website](https://replugged.dev/installation).

### Plugins/Themes
Plugins and themes can be found in our [official Discord server](https://discord.gg/replugged) in #plugin-links and #theme-links, respectively. 
[![Replugged Discord Server](https://discordapp.com/api/guilds/1000926524452647132/widget.png?style=banner2)](https://discord.gg/replugged)

> If the plugin/theme you want isn't in the Replugged server, check out the old archived [Powercord server](https://discord.gg/powercord). Note that some of the plugins may be outdated or abandoned. 

Reviewer: ```suggestion
If the plugin/theme you want isn't in the Replugged server, check out the old [Powercord server](https://discord.gg/powercord). Note that some of the plugins may be outdated or abandoned. 
```
